starlette/testclient.py

```diff
@@ -234,13 +234,17 @@ def __init__(
         raise_server_exceptions: bool = True,
         root_path: str = "",
         *,
+        client: str,
```

Comment:
It should be just a tuple.

```suggestion
        client: tuple[str, int],
```